common/api-review/generative-ai.api.md

```diff
@@ -448,6 +448,13 @@ export interface GenerateContentStreamResult {
     stream: AsyncGenerator<EnhancedGenerateContentResponse>;
 }
 
+// @public
+export interface GeneratedImage {
+    // Warning: (ae-forgotten-export) The symbol "PredictServiceValueType" needs to be exported by the entry point index.d.ts
```

Comment:
Check this file for comments telling you what's still missing in the exported types.

Reply:
Thanks for pointing out. I removed the field from GeneratedImage.